src/interface/src/app/treatments/treatment-map/treatment-map.component.html

```diff
@@ -23,6 +23,15 @@
     *ngIf="showMapControls$ | async"
     [mapLibreMap]="mapLibreMap"></app-map-controls>
 
+  <app-action-button
+    *ngIf="
+      (standSelectionEnabled$ | async) === false &&
```

Comment:
I think we should have the button and legend disappear in stand editing mode, and I think this is the only var that designates that mode. Happy to change this, if there's a better approach.